spacy_llm/tests/pipeline/test_llm.py

```diff
@@ -159,6 +164,9 @@ def parse_responses(
         ) -> Iterable[Doc]:
             return docs
 
+        def prompt_template(self) -> str:
+            return ""
+
```

Comment:
I worry a little here that we're making it more complicated/extensive for users to implement a custom task. Now they'll always have to implement `prompt_template` as well, right?
Isn't there a way we can say that if this property is not available we just assume it to be `""`?

Reply:
Done in https://github.com/explosion/spacy-llm/pull/192/commits/28d6d4b51a6c2c5da8f7a2f89181c5842511d751 and https://github.com/explosion/spacy-llm/pull/192/commits/cd1ccff222bcd498107dd53b5b97b68ed87abb3e.